src/extension.ts

```diff
@@ -1277,7 +1278,7 @@ function createIdfTerminal() {
     const espIdfTerminal = vscode.window.createTerminal({
       name: "ESP-IDF Terminal",
       env: modifiedEnv,
-      cwd: workspaceRoot.fsPath,
+      cwd: workspaceRoot.fsPath || modifiedEnv.IDF_PATH || process.cwd(),
```

Comment:
Why `|| modifiedEnv.IDF_PATH || process.cwd(),`, since we need to launch the monitor in the IDF_Project root, which is `workspaceRoot.fsPath`

Reply:
Fallback when launching a terminal with no opened folder.